build/scripts/publish-docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+
+DIR="${BASH_SOURCE%/*}"
+
+if [ -z "${NETLIFY_BUILD_HOOK_URL}" ] ; then
+  echo "NETLIFY_BUILD_HOOK_URL is required"
+  exit 1
+fi
+
+# shellcheck source=tag-common.sh
+. "${DIR}/tag-common.sh"
```

Comment:
```suggestion
source "${DIR}/tag-common.sh"
```

I like it better when the action is explicit. :)

Reply:
`.` is the portable and POSIX-defined way of sourcing a file. `source`, OTOH, is shell-specific and can be missing in other shells like `dash`. Although the interpreter is set to `bash`, it's best to not rely on non-portable features where possible.